### DISCUSSION TITLE
post/martin-dev-honey-tip-2/

### DISCUSSION THREAD

**Discussion by giscus** [kakaopay/techlog-comments#24] [Announcements]
# post/martin-dev-honey-tip-2/

@Qualifier vs @Primary 누가 이길까?

https://tech.kakaopay.com/post/martin-dev-honey-tip-2/

**Comment by yozmngooddev** [comment]
스프링을 책에 나온데로 사용만 했는데 내부 소스로 설명해주니 넘 재미있네요. 좋은 글 감사합니다.

**Comment by heli-os** [comment]
우와 정성 분석.. 좋은 글 감사합니다.

**Comment by kwj1270** [comment]
궁금증과 해결하기 위해 찾아가는 과정이 재밌네요. 좋은 글 감사합니다.

**Comment by kijung7942** [comment]
주니어 개발자가 보아도 이해하기 쉽게 잘 풀어서 써주셔서 재밌고 유익하게 읽었습니다. 감사합니다!

**Comment by kchung1995** [comment]
글도 재밌고 내용도 읽기 쉽고 유용하네요 ㅎㅎ 좋은 글 감사합니다!

**Comment by TedHong** [comment]
좋은 내용이네요. 잘 보고 갑니다 :)

**Comment by ACACIANh** [comment]
내용도 내용이지만 여러 의견(?)도 재밌었고 개발자로 취해야할 태도도 좋은거같아요 ㅎㅎ

**Comment by duhanmo999** [comment]
재밌고 유익한 글 감사합니다! 빈 주입 우선순위에 대해 잘 알게되었네요 :)

**Comment by meeingjae** [comment]
너무 재밌고 유익한 글이라 저도 모르게 월?루 하며 정독했습니다
글 많이 올려주세요!!! ㅎㅎ
